pkg/common/constants/constants.go

```diff
@@ -56,7 +57,7 @@ const DaemonSetType = "DaemonSet"
 const AppManagerHandlerName = "yunikorn-app"
 
 // Gang scheduling
-const PlaceholderContainerImage = "k8s.gcr.io/pause"
+const PlaceholderContainerImage = "registry.k8s.io/pause:3.7"
```

Comment:
This image is hard-coded at various places, like in the `Dockerfile` or `values.yaml`, do those need to be changed?

Reply:
The other places will be going away once this PR is complete, as we are transitioning to having defaults be in code only.